src/dmd/root/ctfloat.d

```diff
@@ -110,18 +113,23 @@ extern (C++) struct CTFloat
         static real_t copysign(real_t x, real_t s) { return core.stdc.math.copysignl(x, s); }
     }
 
+    pure
     static real_t fmin(real_t x, real_t y) { return x < y ? x : y; }
+    pure
     static real_t fmax(real_t x, real_t y) { return x > y ? x : y; }
 
+    pure
     static real_t fma(real_t x, real_t y, real_t z) { return (x * y) + z; }
 
+    pure @trusted
```

Comment:
Are we putting the attributes on a separate line in other places? I don't think that is particular common to do in D.

Reply:
My assumption is the reason was to minimize the diff

Reply:
that's right. It makes for easy review.

Reply:
@jacob-carlborg The style (attributes on the line before the declaration) is used in Phobos, as well as various other styles. Personally I prefer it, as it looks tidier and more elegant and is easier to keep up with the soft 80 columns rule. Also, I think it is consistent with the majority use of UDAs. Pragmatically speaking, one is allowed to use any style of attributes, until we add a rule to the style guide.